app/components/Sidebar.js

```diff
@@ -2,10 +2,12 @@
 import React, { Component } from 'react';
 import styles from './Sidebar.scss';
 import type { ApplicationData, BaseCacheEntry } from '../reducers/cache';
-import { homepage, version, bugs } from '../../package.json';
+import packageJson from '../../package.json';
```

Comment:
When do you have to remove the implicit destruction and when not? Why here in line 7 you don't do it, but for example for the SENDGRID_API_KEY you have to do it?

Reply:
It's complicated; depends on explicit vs implicit exports – in this case we're importing from JSON (which is experimental anyway) and that has a *default* export consisting of an object, which can be destructured, but not explicit property exports...